plugins/woocommerce-blocks/docs/third-party-developers/extensibility/checkout-block/available-filters/checkout-and-place-order-button.md

```diff
@@ -191,16 +191,16 @@ registerCheckoutFilters( 'example-extension', {
 
 ### Description <!-- omit in toc -->
 
-The `placeOrderButtonLabel` filter allows change the label of the "Place order" button.
+The `placeOrderButtonLabel` filter allows change the label of the "Place order · <price/>" button.
 
 ### Parameters <!-- omit in toc -->
 
--   _defaultValue_ (type: `string`, default: `Place order`) - The label of the "Place order" button.
+-   _defaultValue_ (type: `string`, default: `Place order · <price/>`) - The label of the "Place order · <price/>" button.
 -   _extensions_ `object` (default: `{}`) - The extensions object.
 
 ### Returns <!-- omit in toc -->
 
--   `string` - The label of the "Place order" button.
+-   `string` - The label of the "Place order · <price/>" button.
```

Comment:
```suggestion
-   `string` - The label of the "Place order · &lt;price/>" button.
```

Similar to my previous comments, this part should also be escaped, to ensure that the element is visible, see:

<img width="464" alt="Screenshot 2024-05-16 at 12 13 15" src="https://github.com/woocommerce/woocommerce/assets/3323310/563d8b12-2092-4351-a954-8016d9776a77">